### ISSUE TITLE
Driver Capabilities GKE timesharing

### ISSUE THREAD

**Issue by patrickcorrigan** [NVIDIA/nvidia-docker#1695]
I'm running  `nvidia/cuda:11.8.0-base-ubuntu20.04` on Google Kubernetes Engine using GPU Timesharing on t4 gpus

Checking the driver Capabilities I get compute and utility. I was hoping to also get graphics and video. Is this a limitation of Timesharing on GKE?

**Comment by elezar** [comment]
You need to set the `NVIDIA_DRIVER_CAPABILITIES=all` in the container that you are starting. Just a note, if you're using the  Google Device plugin, then there is no support for `NVIDIA_DRIVER_CAPABILITIES` at all, and changing this should not affect the libraries / binaries that are included.

Is there something that you're trying to do in the container that you cannot?

**Comment by patrickcorrigan** [comment]
Hi @elezar, 

I'm currently running containers on a cluster where I start X Window sessions, run graphical applications, capture the display, encode it to h.264 and stream it over WebRTC to users. Unfortunately I'm doing all of this on the CPU 🥲 Which doesn't lead to great performance for more intense work loads. 

I was hoping I could add in some T4 gpu nodes for a boost in rendering performance and to use the NVEC encoder for faster h.264 encoding 🚀

I wanted to share the T4 between 4 containers to make it more economical to run 💵

I can get the pods up and running no problem. I can start an X Session using a 'xf86-video-dummy' like I used to when using a CPU only machine. When I run applications then it doesn't seem to use the GPU at all 🐢

So I used `nvidia-xconfig` to create a x config with a dummy display and specified the `nvidia` driver. Unfortunately I get 

```
Failed to module 'nvidia' (Module does not exist, 0)
```

I was wondering if I needed to install the gpu drivers on the container machine too and that's what lead me to here 👋 and this was the most isolated question I could think of asking.

TL;DR 📖
Running hardware accelerated X Sessions, capturing and then encoding the output using NVEC encoder and then streaming over WebRTC.

Thank your for your quick response
